docs/user-guide/localizing-your-theme.md

```diff
@@ -21,6 +21,14 @@ enable `i18n` (internationalization) support on MkDocs:
 pip install mkdocs[i18n]
 ```
 
+## Supported languages
+
+MkDocs provided themes `mkdocs` and `readthedocs` currently support the
+following `theme.locale` languages:
+
+- `en`: English
+- `fr`: French
+
```

Comment:
Adding this information here assumes that the translations for both themes will always be the same. While that would certainly be preferred, we cannot assume that a translator will always provide a complete translation for both themes. Some users may only be willing to spend time on the one they use.

In fact, that is one of the reasons why I changed all of the examples in the docs to only show one theme (the other was to keep the instructions simple). If/when the list diverges for each theme, it will no longer make sense for them to be listed here. Instead, they should be listed under the `locale` config option for each theme in [choosing-your-theme.md][1].

I suppose we could probably include a link here which points to each and informs users that they can find a list there.

[1]: https://github.com/mkdocs/mkdocs/pull/2299/files#diff-a1c2b3e8d0dd115bd0895b9e881dfe8ba432ee745726a9b0e49093fb7a68bdde

Reply:
> From our conversation history we agreed that mkdocs provided themes should support the same languages so I just pushed a section in the user guide theme localization with a list of supported languages.

Ok I thought otherwise. I updated the documentation as such then.